utils/api.ts

```diff
@@ -0,0 +1,8 @@
+export type APIWaste = {
+  id: number;
+  description: string;
+  typeof: "rest" | "recycle" | "paper" | "bio";
+  image: string;
```

Comment:
`image` could be a base64 encoded string but you like to set the path/src of it. Maybe rename to `imageSrc`?